pynav/filters.py

```diff
@@ -104,7 +105,10 @@ def predict(
             x_new.symmetrize()
             x_new.state.stamp = x.state.stamp + dt
 
-        return x_new
+        if output_jac:
+            return x_new, A
+        else:
+            return x_new
```

Comment:
not urgent, since this is a non-breaking change for others, but i recommend we implement something similar to the correction step both for consistency and since it is an easily expandable solution